app/common/directives/mode-bar/mode-bar.html

```diff
@@ -32,7 +32,7 @@
             </li>
 
             <li>
-                <a class="more-menu-trigger" ng-class="{ active : moreActive }" ng-click="showMore($event)">
+                <a href="" class="more-menu-trigger" ng-class="{ active : moreActive }" ng-click="showMore($event)">
```

Comment:
Hey @Ifycode I don't know if it is the right way to enable navigation. Can you see if we can do it  by setting the `tabindex` instead as it might be the suggested way to enable navigation.
You can read more about tabindex here: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Maybe @Angamanga and @rowasc have more to add to this.

Reply:
@renujain31 Thank you noticing and reviewing 🎉

Reply:
Ok. Please see comment https://github.com/ushahidi/platform-client/pull/1564#issuecomment-713254498